content/blog/project-management-for-individuals/index.md

```diff
@@ -0,0 +1,134 @@
+
+So I developed the most simple thing that would work.
+
+### The task types
+
+There are 2 task types:
+
+* Tasks: You can nest tasks within tasks. They can be anything: feature implementation, refactoring, writing tests, chores, etc. Anything except issues or bugs
+
+* Bugs: They can be nested under tasks, but not under other bugs. They must be atomic, meaning they can't be broken down into smaller pieces
+
+### Tags
+
+There are a few tags that I like to use when defining bugs and tasks:
+
+* Feature: For tasks the implement a feature
+
+* Refactor: For tasks that relate to a refactor of some sorts
+
+* Chore: For tasks that may be boring / don't affect the end result too much (documentation)
+
+* Test: For tasks or bugs about testing
+
+* Blocker: For bugs that are very urgent
+
+By using these tags it becomes easier to manage tasks and bugs as you can filter through them. Some people might prefer making each tag it's own task type, but personally I prefer to have the one type of task with one or more tags instead.
+
+### Branching
+
+For branching I use the following method:
+
+1. Create a task that the branch relates to
+
+2. Create a branch with a brief description of what it's for and the task / bug number, e.g: 372-fix-document-view
+
+3. Make your changes
+
+4. Push your changes
+
+5. Create a pull request
+
+6. Merge with the main or trunk branch
+
+7. Execute CI/CD operations to test and build the project
+
+8. If all is successful, merge main and production branches
+
+### Development Cycle
+
+The development cycle, for those that don't know, is the process followed by an individual or team to develop software.
+
+This is what my usual development cycle looks like:
+
+1. Requirements: Firstly I gather the requirements of the product or feature
+
+2. Design: I usually jot down on a notebook a rough design of the feature. Design here doesn't refer exclusively to UI, but also code design. How will I break this down? If I'm in the backend, do I need a new controller? Do I need to add a service?
+
+3. Writing tests: I prefer test driven development, so firstly I write unit and integration tests where I can. And I make sure they all fail!
+
+4. Implementation: Now that I've got the tests, I will program my implementation. For every test that succeeds I will create a commit.
+
+5. Refactoring / Documenting: Now that it's implemented and working, I do some refactoring and documenting where needed to make it easier to understand.
+
+6. CI: Now I will push all of my commits, and wait for the Continuous Integration pipeline to verify that all the testing, linting and building is successful
+
+7. (Optional) CD: This step is optional because I don't always deploy the changes I make immediately. If I'm fixing an important bug then this step will take place immediately after CI. If not then I'll wait to accumulate a few features to deploy my updated application
```

Comment:
I think it'd be great if you'd say Continuous Delivery somewhere in here, like in the CI section.